b2sdk/raw_simulator.py

```diff
@@ -1038,8 +1033,7 @@ def list_buckets(
 
         # Do the query
         sorted_buckets = [
-            self.bucket_name_to_bucket[name]
-            for name in sorted(six.iterkeys(self.bucket_name_to_bucket))
+            self.bucket_name_to_bucket[name] for name in sorted(self.bucket_name_to_bucket)
```

Comment:
wouldn't sorting output of `.items()` be faster here due to not needing to first go through N to get every key and then retrieving every value with the theoretical "O(1)" which really is O(log N)? I mean, there's a max of 100 entries in this thing anyway so maybe it doesn't matter much, but I'm wondering why you chose to write it this way

Reply:
I didn't choose to write it this way :) For most parts, I used regexes and structural replacement to get rid of six. Then I did a review for myself without analyzing the code for quality and speed, only for correctness.

I will update it

Reply:
It did what six is doing under the hood for Python 3 with few exceptions, so it didn't introduce any regression.

Reply:
Using `items()` is actually 2x slower because of tuple lookups and unpacking

Reply:
🤦